frontend/apps/picasso/defi/polkadot/pallets/Assets/ExistentialDeposits/karura.ts

```diff
@@ -3,15 +3,16 @@ import BigNumber from "bignumber.js";
 
 // @see https://wiki.acala.network/get-started/get-started/karura-account
 const karuraExistentialDeposits: {
-    [key in string] : BigNumber
+  [key in string]: BigNumber;
 } = {
-    "kusd": new BigNumber(0.01),
-    "ausd": new BigNumber(0.01),
-    "kar": new BigNumber(0.1),
-    "ksm": new BigNumber(0.0001),
-}
-
+  kusd: new BigNumber(0.01),
+  ausd: new BigNumber(0.01),
+  kar: new BigNumber(0.1),
+  ksm: new BigNumber(0.0001),
+};
 
 export const getKaruraExistentialDeposit = (tokenId: TokenId): BigNumber => {
-    return tokenId in karuraExistentialDeposits ? karuraExistentialDeposits[tokenId] : new BigNumber(1)
-}
+  return tokenId in karuraExistentialDeposits
+    ? karuraExistentialDeposits[tokenId]
+    : new BigNumber(1);
```

Comment:
@easteregg shouldnt it be raised to 10^12?

Reply:
No.